hack/update-openapi-spec.sh

```diff
@@ -77,10 +77,11 @@ kube::log::status "Starting kube-apiserver"
 APISERVER_PID=$!
 
 if ! kube::util::wait_for_url "${API_HOST}:${API_PORT}/healthz" "apiserver: "; then
-  kube::log::error "Here are the last 10 lines from kube-apiserver (${API_LOGFILE})"
-  kube::log::error "=== BEGIN OF LOG ==="
+  kube::log::error "Please refer to kube-apiserver log (${API_LOGFILE}) for more details."
+  kube::log::info "Here are the last 10 lines from kube-apiserver (${API_LOGFILE})"
+  kube::log::info "=== BEGIN OF LOG ==="
   tail -10 "${API_LOGFILE}" || :
```

Comment:
wouldn't it make more sense to direct this to stderr?

Reply:
Do you mean `tail -10 "${API_LOGFILE}" >> &2`?

I will have a try.

Reply:
Yeah! It works!
```
Running update-openapi-spec
!!! [1227 09:15:09] Timed out waiting for apiserver:  to answer at 127.0.0.1:8050/healthz; tried 30 waiting 1 between each
!!! [1227 09:15:09] Here are the last 10 lines from kube-apiserver (/tmp/openapi-api-server.log)
!!! [1227 09:15:09] === BEGIN OF LOG ===
      --log-file-max-size uint           Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited. (default 1800)
      --log-flush-frequency duration     Maximum number of seconds between log flushes (default 5s)
      --logtostderr                      log to standard error instead of files (default true)
      --skip-headers                     If true, avoid header prefixes in the log messages
      --skip-log-headers                 If true, avoid headers when opening log files
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          number for the log level verbosity (default 0)
      --version version[=true]           Print version information and quit
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

!!! [1227 09:15:09] === END OF LOG ===
!!! [1227 09:15:09] Call tree:
!!! [1227 09:15:09]  1: /root/horen/go/src/k8s.io/kubernetes/hack/update-openapi-spec.sh:1 cleanup(...)
```

Code change is:
```diff
@@ -79,7 +79,7 @@ APISERVER_PID=$!
 if ! kube::util::wait_for_url "${API_HOST}:${API_PORT}/healthz" "apiserver: "; then
   kube::log::error "Here are the last 10 lines from kube-apiserver (${API_LOGFILE})"
   kube::log::error "=== BEGIN OF LOG ==="
-  tail -10 "${API_LOGFILE}" || :
+  tail -10 "${API_LOGFILE}" >&2 || :
   kube::log::error "=== END OF LOG ==="
   exit 1
 fi
```